src/w3c/templates/sotd.js

```diff
@@ -439,22 +449,58 @@ function linkToWorkingGroup(conf) {
   if (!conf.wg) {
     return;
   }
-  let proposedChanges = null;
+  let changes = null;
   if (conf.isRec && conf.revisionTypes && conf.revisionTypes.length) {
-    if (conf.revisionTypes.includes("addition")) {
-      if (conf.revisionTypes.includes("correction")) {
-        proposedChanges = html`It includes
-          <a href="${processLink}#proposed-amendments">proposed amendments</a>,
-          introducing substantive changes and new features since the previous
-          Recommendation.`;
-      } else {
-        proposedChanges = html`It includes
-          <a href="${processLink}#proposed-addition">proposed additions</a>,
-          introducing new features since the previous Recommendation.`;
-      }
-    } else if (conf.revisionTypes.includes("correction")) {
-      proposedChanges = html`It includes
-        <a href="${processLink}#proposed-correction">proposed corrections</a>.`;
+    // if (conf.revisionTypes.includes("proposed-addition")) {
+    //   if (conf.revisionTypes.includes("proposed-correction")) {
+    //     proposedChanges = html`It includes
+    //       <a href="${processLink}#proposed-amendments">proposed amendments</a>,
+    //       introducing substantive changes and new features since the previous
+    //       Recommendation.`;
+    //   } else {
+    //     proposedChanges = html`It includes
+    //       <a href="${processLink}#proposed-addition">proposed additions</a>,
+    //       introducing new features since the previous Recommendation.`;
+    //   }
+    // } else if (conf.revisionTypes.includes("proposed-correction")) {
+    //   proposedChanges = html`It includes
+    //     <a href="${processLink}#proposed-correction">proposed corrections</a>.`;
+    // }
```

Comment:
Can remove this, I guess?

Reply:
Indeed. Done in https://github.com/w3c/respec/pull/4273/commits/5a31c3e25c890b2d66299c68b965c3b35c0e6fb1